consensus/src/experimental/buffer_manager.rs

```diff
@@ -39,21 +39,28 @@ pub struct OrderedBlocks {
 }
 
 /*
- *                       LedgerInfo Buffer
- *
- *                      ┌────────────┐                    ┌────────────┐
+ *                                                         ┌─────────┐
+ *                                                         │ Signing │
+ *                                                         │ Root    │
+ *                                                         └────┬────┘
```

Comment:
I thought signing root is always the aggregation root?

Reply:
I intend to allow concurrent signature aggregation (as long as the target ledger info enters the buffer, we should keep the signature received from the commit vote).